### DISCUSSION TITLE
Audit should report unknown/unsupported step locations

### DISCUSSION THREAD

**Discussion by kenmuse** [github/gh-actions-importer#137] [Feedback]
When doing an audit, GAI reports build steps that are known, unknown, and unsupported. For unknown and unsupported steps, it would be helpful to have a report similar to `workflow_usage.csv` that contains details about which pipelines/workflows contain those steps. Having this would make it easier to locate those steps, identify the team(s) responsible, and coordinate on an appropriate transformer.

**Comment by tomswilliams** [comment]
@manishapriya94 - fyi for Two Sigma
